src/WalkingModule/src/Module.cpp

```diff
@@ -796,15 +979,66 @@ bool WalkingModule::updateModule()
 
             auto leftFoot = m_FKSolver->getLeftFootToWorldTransform();
             auto rightFoot = m_FKSolver->getRightFootToWorldTransform();
-            m_walkingLogger->sendData(m_FKSolver->getDCM(), m_DCMPositionDesired.front(), m_DCMVelocityDesired.front(),
+
+            iDynTree::Vector2 DCMError;
+            iDynTree::toEigen( DCMError)=iDynTree::toEigen(m_DCMPositionSmoothed)- iDynTree::toEigen( m_dcmEstimatedI);
```

Comment:
Try to be compliant with the stile of the code
```suggestion
            iDynTree::toEigen(DCMError) = iDynTree::toEigen(m_DCMPositionSmoothed)- iDynTree::toEigen( m_dcmEstimatedI);
```

Reply:
solved!